### DISCUSSION TITLE
Moving to OIDC

### DISCUSSION THREAD

**Discussion by razor950** [outline/outline#6274] [Self Hosting]
Hey, if outline has been rolled out initially by slack / google or another authenticator system like that and one then moves over to OIDC. Is there anything one is supposed to do to allow both to work or am I supposed to make sure the other system isn't on?

I say this because I am having weird issues with OIDC <-> outline. I'll login with my info and get in a login loop but if I use the same info on another browser everything is fine.

I have cleared cookies of my OIDC (Authentik) and cleared any outline cookies as well on the browser with issues. I've also revoked all tokens granted by Authentik to outline. 

Edited: Forgot to add that clicking the continue with email, it used to send me a magic link but now when I do that step, it routes to my OIDC provider and causes a login loop as well.

**Comment by tommoor** [comment]
> I say this because I am having weird issues with OIDC <-> outline. I'll login with my info and get in a login loop but if I use the same info on another browser everything is fine.

I'm not sure about this, nothing comes to mind.

> Is there anything one is supposed to do to allow both to work or am I supposed to make sure the other system isn't on?

You can have multiple auth providers, if the user has the same email address they will be logged into the same account. If it is different then a new account will be created for each auth provider.

> clicking the continue with email, it used to send me a magic link but now when I do that step, it routes to my OIDC provider

This is a security measure to prevent skipping any 2FA measures your SSO provider might have.
